main.go

```diff
@@ -4,53 +4,47 @@ import (
 	"flag"
 	"github.com/projectcalico/k8s-policy/pkg/controllers/namespace"
 	"github.com/projectcalico/k8s-policy/pkg/controllers/pod"
+	"github.com/projectcalico/k8s-policy/pkg/controllers/networkpolicy"
+	"github.com/projectcalico/k8s-policy/pkg/config"
 	"github.com/projectcalico/libcalico-go/lib/client"
 	log "github.com/sirupsen/logrus"
 	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	"k8s.io/client-go/tools/clientcmd"
-	"os"
 )
 
 func main() {
-	logLevel, err := log.ParseLevel(os.Getenv("LOG_LEVEL"))
+	
+	config := new(config.Config)
+	err := config.Parse()
 	if err != nil {
-
-		// Defaulting log level to INFO
+		panic(err.Error())
```

Comment:
This should be `log.WithErr(err).Fatal("Failed to parse config")` rather than just a `panic`